### PR TITLE
switch to py3

### PR DESCRIPTION
**Pull request by kshtsk**

### CODE REVIEW

**Comment by djgalloway** [comment]
I'll have to defer to @tchaikov or maybe @zmc on this.  Every time I approve a python PR, it bites me.

**Comment by kshtsk** [comment]
thanks @djgalloway I hope it won't bite, since we merge the corresponding ceph/ceph-cm-ansible#603 will see what @tchaikov says, meanwhile can try and test this PR on sepia?

**Comment by kshtsk** [comment]
Because the ceph-cm-ansbile depends on this PR I guess for pulpito because of py3, so we need both

File: pulpito/controllers/job.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 from pecan import expose
 from pecan import conf
 import requests
-import urlparse
-from util import prettify_job
+from urllib.parse import urljoin
```
Comment (tchaikov) [inline pulpito/controllers/job.py:4]: so I assume that we are dropping the py2 support.

**Review by tchaikov — APPROVED** [review]
LGTM. but, @kshtsk, how did you test this change? as the travis based test basically only builds, run a docker image and check if the image is up and running. so it does not imply that the change works with python3, IMHO.

File: pulpito/controllers/job.py (hunk 1 above)
Comment (kshtsk) [reply pulpito/controllers/job.py:4]: Yes, we drop it here.

**Comment by kshtsk** [comment]
> LGTM. but, @kshtsk, how did you test this change? as the travis based test basically only builds, runs a docker image and checks if the image is up and running. so it does not imply that the change works with python3, IMHO.

I have tested it using manual run of teuthology-deploy pointing the pulpito, paddles, and ceph-cm-ansible repos/branches.

**Pull request merged**